plugin/src/main/java/com/denizenscript/denizen/events/block/BrewsScriptEvent.java

```diff
@@ -25,11 +28,22 @@ public class BrewsScriptEvent extends BukkitScriptEvent implements Listener {
     // @Context
     // <context.location> returns the LocationTag of the brewing stand.
     // <context.inventory> returns the InventoryTag of the brewing stand's contents.
+    // <context.fuel_level> returns an ElementTag(Number) of the brewing stand's fuel level.
+    // <context.result> returns a ListTag(ItemTag) of the brewing stand's result.
+    //
+    // @Determine
+    // "RESULT:<ListTag(ItemTag)>" to change the item that is brewed.
     //
     // -->
 
     public BrewsScriptEvent() {
         registerCouldMatcher("brewing stand brews");
+        this.<BrewsScriptEvent, ListTag>registerDetermination("result", ListTag.class, (evt, context, listTag) -> {

```

Comment:
Should probably have a more descriptive name rather then just `listTag` (e.g. `result`)